src/Core/Makefile.am

```diff
@@ -7,3 +13,10 @@ SUBDIRS = \
 		FSpot.Query \
 		FSpot.Gui \
 		FSpot.Platform
+
+RESOURCES = \
+		FSpot.Core.UnitTest/TestData/taglib-sample.xmp \
+		FSpot.Core.UnitTest/TestData/taglib-sample-broken.xmp \
+		FSpot.Core.UnitTest/TestData/taglib-sample.jpg \
+		FSpot.Core.UnitTest/TestData/taglib-sample-broken.jpg
+		
```

Comment:
I think the test images need to be in the distribution tar ball in order
for the unit tests to run. There may be a better way of achieving this than
adding them to RESOURCES? I'm not an Automake expert I'm afraid.

Christopher

On 16 June 2015 at 07:05, Stephen Shaw <notifications@github.com> wrote:

> In src/Core/Makefile.am
> <https://github.com/mono/f-spot/pull/29#discussion_r32493263>:
>
> > @@ -7,3 +13,10 @@ SUBDIRS = \
> >  		FSpot.Query \
> >  		FSpot.Gui \
> >  		FSpot.Platform
> > +
> > +RESOURCES = \
> > +		FSpot.Core.UnitTest/TestData/taglib-sample.xmp \
> > +		FSpot.Core.UnitTest/TestData/taglib-sample-broken.xmp \
> > +		FSpot.Core.UnitTest/TestData/taglib-sample.jpg \
> > +		FSpot.Core.UnitTest/TestData/taglib-sample-broken.jpg
> > +		
>
> I'm not sure that we want to include test data in 'RESOURCES'?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mono/f-spot/pull/29/files#r32493263>.
>